test/ecto/query/planner_test.exs

```diff
@@ -465,6 +465,27 @@ defmodule Ecto.Query.PlannerTest do
     assert [:all, {"comments", Comment, _, nil}, {:recursive_cte, "cte", ^expr}] = cache
   end
 
+  test "prepare: Prepare Update CTE" do
```

Comment:
This test was to help reproduce the issue I was having with params missing from CTE expressions.
Not sure if it is worth keeping?

Reply:
Yes, it is worth keeping a test with CTE but we should rewrite it to do similar assertions to the test above. also please fix the casing in the test title. :)

Reply:
👍 I've added assertions for the structure of the query and cache key.